usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -7369,6 +8248,8 @@ static int read_adapter_config_file(STDLL_TokData_t * tokdata,
                                31) == 0) {
                i = 0;
                ep11_data->optimize_single_ops = 1;
+            } else if (strncmp(token, "PKEY_MODE", 9) == 0) {
+               i = 6;
```

Comment:
In the final ese case: Don't you need to update the error message (trace + syslog) with the new keyword you are adding?
I.e. "Expected ..... <your new keyword>, found ...."

Reply:
Puh ... we really list all the keywords ... yes, in this case PKEY_MODE should also be listed. Done.